### ISSUE TITLE
package.json typo and build requirements

### ISSUE THREAD

**Issue by AdditionAddict** [Acaeris/draconika#1]
`"postinstall": "npm run postinstall:electron && electron-builder install-app-deps",`
` ___________________________^`

Also needs devDependancy
`"@types/jquery": "^3.3.30",`
`npm install --save-dev @types/jquery`

angular.json
```
            "scripts": [
              "node_modules/jquery/dist/jquery.min.js"
            ]
```

src\app\directives\dropdown.directive.ts
`import * as $ from "jquery";`

**Comment by Acaeris** [comment]
Thank you. I had fixed some of these earlier. I have added the script to the angular.json. I haven't directly included jQuery in the directive as that actually breaks the functionality of the Fomantic dropdowns due to their extension of jQuery not being supported that way. Without the import, it does throw an error in console but the dropdowns function correctly. Not sure of the best solution to that.
